app/code/local/Magmodules/Webwinkelconnect/Helper/Data.php

```diff
@@ -97,7 +97,133 @@ public function isProductReviewInviteEnabled(): bool {
             return false;
         }
 
+        $rating_options_collection = Mage::getModel('rating/rating')->getCollection();
+        if ($rating_options_collection->getSize() < 1) {
+            Mage::throwException($this->__('You have no ratings created. Please create at least one rating to use the product reviews feature.'));
```

Comment:
Better use `throw` directly, so that static analysis tools and your IDE understand that the execution flow will never pass this line.

```suggestion
            throw Mage::exception('Magmodules_Webwinkelconnect', $this->__('You have no ratings created. Please create at least one rating to use the product reviews feature.'));
```

Reply:
The problem here is that if an exception is thrown by using `throw Mage::exception` the config page blows up with error 500

Reply:
Is that because of the exception class, ie, does it work with `Mage_Core` instead of `Magmodules_Webwinkelconnect`? Because `Mage::throwException($message)` just does `throw new Mage_Core_Exception($message)`, so it shouldn't make a difference.